ckanext/data_qld/ckan_dataset.json

```diff
@@ -229,8 +230,7 @@
       "display_snippet": null,
       "help_text": "This dataset’s ‘Next update due’ date is automatically updated when ‘Adding new time series data to the resource’ is selected. This date can be subsequently edited in the dataset ‘Manage’ page.",
       "validators": "data_qld_validate_nature_of_change_data ignore_empty scheming_choices",
-      "create_validators": "ignore",
-      "choices":[
+      "choices": [
```

Comment:
Is this also required to be added to the geo science plugin schema?
https://github.com/qld-gov-au/ckanext-harvester-data-qld-geoscience/blob/develop/ckanext/harvester_data_qld_geoscience/geoscience_dataset.json

Reply:
The geoscience harvester does not import any resources and has no resources metadata field in the schema, so I think we are good.
Also, all mandatory values in the schema are set via the harvest config and it also does not use any `data_qld` validators so it can be used without that extension.